tests/test_tpt_reports.py

```diff
@@ -11,33 +11,26 @@
 import pytest
 
 # cisagov Libraries
-import example
-
-div_params = [
-    (1, 1, 1),
-    (2, 2, 1),
-    (0, 1, 0),
-    (8, 2, 4),
-]
+import tpt_reports
 
 log_levels = (
     "debug",
     "info",
-    "warning",
+    # "warning",
```

Comment:
Why is `"warning"` commented out here?

Reply:
This was just a test we were doing locally and was missed on pushing changes. It has been added back.
Resolved in 7183dd5